src/content/docs/it/tutorial/1-setup/4.mdx

```diff
@@ -0,0 +1,71 @@
+---
+type: tutorial
+title: Salva il tuo codice in un repository online
+description: |-
+  Tutorial: Costruisci il tuo primo blog Astro —
+  Crea un repository GitHub per il progetto del tutorial
+i18nReady: true
```

Comment:
```suggestion

```

This is just for the primary version... it's not really clear from the variable itself.